src/display.rs

```diff
@@ -181,8 +181,8 @@ impl Display {
         window.set_inner_size(viewport_size.to_logical(dpr));
         renderer.resize(viewport_size, padding_x as f32, padding_y as f32);
 
-        info!("Cell Size: ({} x {})", cell_width, cell_height);
-        info!("Padding: ({} x {})", padding_x, padding_y);
+        trace!("cell size: ({} x {})", cell_width, cell_height);
+        trace!("padding: ({} x {})", padding_x, padding_y);
```

Comment:
These are very important log messages for debugging problems with the dimensions, so I'd keep them in the `info!` level. Only with padding, cell size and window dimensions, it's possible to make sure everything was calculated properly.

Reply:
Seems like they should be `debug!` then? Is the main concern just having them in the output by default?

Would setting the default log level to debug be a better solution?

Reply:
I think the info level is fine here. The user might be interested in stuff like that. So you know the real padding after DPI scaling is applied and stuff like that.

Having a fixed definition for which log levels contain exactly which messages is a different story though. We could also handle that in this PR, but I don't think the issue is bad enough to re-check every log level now. So I wouldn't change them.